### DISCUSSION TITLE
Remove Blue Screen instead of Green Screen

### DISCUSSION THREAD

**Discussion by williamfrantz** [ImageMagick/ImageMagick#5977] [Digital Image Processing]
How can I modify "greenscreen" from Fred's ImageMagick Scripts, to remove a blue screen instead of a green screen?
http://www.fmwconcepts.com/imagemagick/greenscreen/index.php

There's a section titled, "# do modulatecolor2 processing to desaturate green" and it has a variable defined as "hue=120" which seems to be the value of green, but simply changing this to "hue=240" (blue) doesn't seem to do the trick.

**Comment by fmw42** [comment]
Post an example image.  Please contact me offline and I will look into it. (fmw at alink dot net)

**Comment by williamfrantz** [comment]
![frame-00100](https://user-images.githubusercontent.com/4675630/212567354-692a94c5-5856-40c4-bae2-1224093489b3.jpg)
Attached is an example image and I sent an email to "fmw".
